dojo/templates/dojo/view_test.html

```diff
@@ -678,6 +681,7 @@ <h4 class="has-filters"> Findings ({{findings.total_count}}) <small>{{ test.id|g
                             {% if 'is_finding_groups_enabled'|system_setting_enabled %}
                                 <th>Group</th>
                             {% endif %}
+                            <th>Planned Remediation Date</th>
```

Comment:
The list has already a lot of fields and this title needs much more space than the actual date. Can we just write "Planned Remediation"?